### DISCUSSION TITLE
How to setup OnlyOffice?

### DISCUSSION THREAD

**Discussion by JeCheeseSmith** [nextcloud/all-in-one#5302] [Questions] [answered]
Hi, I found out that there's still packages being updated for the an only-office container.? (See https://github.com/nextcloud/all-in-one/tree/main/Containers/onlyoffice & https://hub.docker.com/r/nextcloud/aio-onlyoffice)

Looking through older questions it seems to has dropped support? Is that so? Or is there still a way to enable it?

 For school we regularly uses .pptx & .dockx files., so it would be nice to be able to edit some of them in Nextcloud AIO as well!
 
 This [issue](https://github.com/nextcloud/all-in-one/discussions/1140) hinted to configuration.json, where I do find an Onlyoffice Key and the value: isOnlyofficeEnabled = 0. Would setting this to true enable support for the container?

**Comment by szaimen** [comment, marked as answer]
Hi, see https://github.com/nextcloud/all-in-one/discussions/2461
